### PR TITLE
Implement means for SaveACopyAs to open the copy post-save

### LINKED ISSUE #11861: [Feature Request] Save a Copy As and then OPEN THE COPY
Often I need to copy a file that I have open in Notepad++ and then edit that file immediately, while keeping the original file open as well (for reference or independent change).

This becomes an operation of multiple steps because neither _Save As..._ or _Save a Copy As..._ will conclude with both the original and the copy open in Notepad++ tabs.

So I would like to see a way to achieve this in one step, e.g. a new command called _Save a Copy As and Open The Copy..._.

### Debug Information

Above comments based on behavior current to Notepad++ 8.4.2

### Workarounds

Currently I work-around the (minor) pain of the missing feature by:

* Using _Save As_ and then _File Open_ and then find my original file and reopen it
* Using _Save a Copy As_ and then _File Open_ and then find my new file and open it
* Getting an Exporer window in the parent folder of the original file, use drag-n-drop copy of the file, rename the file, and then drag-n-drop the renamed file into Notepad++

### PR DESCRIPTION
**Pull request by alankilborn**
Fix #11861

### CODE REVIEW

**Comment by MarkusBodensee** [comment]
And futher comments there in PR #15413

File: PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp [hunk 1]
```diff
@@ -902,6 +927,8 @@ class CustomFileDialog::Impl
 	bool _enableFileTypeCheckbox = false;
 	bool _fileTypeCheckboxValue = false;	// initial value
 	wstring _fileTypeCheckboxLabel;
+	bool _isSavingAsCopy = false;
+	bool _isOpeningTheCopy = false;
```
Comment (donho) [inline PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp:931]: https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15413#discussion_r1678674104

**Review by donho — CHANGES_REQUESTED** [review]
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15413#discussion_r1678674104

File: PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp (hunk 1 above)
Comment (alankilborn) [reply PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp:931]: Oops, missed deletion.  :-(

Comment (alankilborn) [reply PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp:931]: Hmm, something bigger went wrong.  :-(

**Review by donho — APPROVED** [review]

**Pull request closed without merging**